src/pybind/mgr/volumes/fs/async_job.py

```diff
@@ -136,11 +136,31 @@ def __init__(self, volume_client, name_pfx, nr_concurrent_jobs):
         self.wakeup_timeout = None
 
         self.threads = []
-        for i in range(self.nr_concurrent_jobs):
-            self.threads.append(JobThread(self, volume_client, name="{0}.{1}".format(self.name_pfx, i)))
-            self.threads[-1].start()
+        self.spawn_all_threads()
         self.start()
 
+    def spawn_new_thread(self, suffix):
+        t_name = f'{self.name_pfx}.{time.time()}.{suffix}'
+        log.info(f'spawning new thread with name {t_name}')
```

Comment:
why are these in info log level? They should be at debug level IMO.

Reply:
This one is pretty useful bit of information, it should be an info level IMO.

Reply:
It just tells it's spawning a thread. INFO should be used when you dump something like counters, etc. which helps in understanding how those values change over time.

Reply:
Done. PTAL.